src/tokenize.cpp

```diff
@@ -72,28 +72,28 @@ struct tok_ctx
 
    bool more()
    {
-      return(c.idx < (int)data.size());
+      return(c.idx < data.size());
    }
 
 
-   int peek()
+   size_t peek()
    {
-      return(more() ? data[c.idx] : -1);
+      return(more() ? data[c.idx] : 0);
    }
 
 
-   int peek(int idx)
+   size_t peek(size_t idx)
    {
       idx += c.idx;
-      return((idx < (int)data.size()) ? data[idx] : -1);
+      return((idx < data.size()) ? data[idx] : 0);
    }
 
 
-   int get()
+   size_t get()
```

Comment:
The function `get()` returns an element of data which has type `int`. Is it really wanted to convert the data to `size_t`?
In this case why is the data array not also changed to type `size_t` too?
Would it not be better to use `UINT32` for the data array?